src/types/position.ts

```diff
@@ -22,6 +26,16 @@ export class AbsolutePosition {
     minWidth = "",
     maxWidth = ""
   ) {
+    if (
+      invalidSize(x) ||
+      invalidSize(y) ||
+      invalidSize(width) ||
+      invalidSize(height)
+    ) {
+      throw new Error(
+        `Invalid AbsolutePosition (${x},${y},${width},${height})`
+      );
+    }
```

Comment:
This bit seems to work well to pick up errors in the file at runtime 😄

Reply:
That's the intention. More generally, if we can do validation like this then we have a bit more control.